vagrant/provisioning/roles/arkcase-file-management/tasks/main.yml

```diff
@@ -1,7 +1,37 @@
+- name: install required packages
+  become: yes
+  yum:
+    state: installed
+    name: python3-pip
+  vars:
+    ansible_python_interpreter: /usr/bin/python2
+
+- name: install Python boto3 library
+  become: yes
+  pip:
+    executable: pip3
+    name: ['boto3', 'botocore']
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+
+- name: Create buckets
+  become: yes
+  s3_bucket:
+    s3_url: "{{ file_management_s3_url }}"
+    aws_access_key: "{{ file_management_s3_access_key }}"
+    aws_secret_key: "{{ file_management_s3_secret_key }}"
+    name: "{{ item }}"
+    state: present
+  with_items: "{{ file_management_s3_buckets }}"
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
```

Comment:
The amazon.aws collection requires python 3.6 and above. So, you will notice that I have to put python version to use almost on all tasks.